ceph_deploy/mds.py

```diff
@@ -138,6 +138,11 @@ def mds_create(args):
                 distro.release,
                 distro.codename
             )
+            if distro.normalized_name == 'centos':
```

Comment:
I'm not sure I follow the ``isntall-mds`` logic.  What would this flag apply to, ``ceph-deploy mds create``, or ``ceph-deploy install``?

Truthfully, I feel like no matter which way we approach it, it's going to be a way that effects the default behavior for upstream/community users.  I don't think the change will be backwards compatible.  We have a an idea of how to achieve that, though, which is the modularization you mentioned.  I would suggest waiting for that?

The BZ that brought this up isn't an immediate priority that I can tell.  I think it can wait to be done right.  What are your thoughts?